lib/matplotlib/backends/backend_qt5agg.py

```diff
@@ -134,7 +136,7 @@ def blit(self, bbox=None):
         """
         # If bbox is None, blit the entire canvas. Otherwise
         # blit only the area defined by the bbox.
-        if bbox is None and self.figure:
+        if bbox is None:
             bbox = self.figure.bbox
 
         self._bbox_queue.append(bbox)
```

Comment:
That would technically be backwards incompatible (and the behavior is "mandated" by the docstring of `FigureCanvasBase`), but a quick grep suggests that we never rely on it (always explicitly passing in the figure bbox when needed) so I'm fine with getting rid of the behavior.  You get to decide on the deprecation policy :-)